.github/workflows/ci.yml

```diff
@@ -66,6 +85,7 @@ jobs:
           architecture: "x64"
       - run: pip install flake8 flake8-print
       - run: flake8 --enable=T modin/ asv_bench/benchmarks
+      - run: flake8 scripts/
```

Comment:
I see that this is separate to avoid the `print` checks because there is a print in the test. Is it possible to use `logging` instead of print below?

Reply:
In the latest version I use `logging`, please review.